### PR TITLE
Changes - Profile Design

### PR DESCRIPTION
**Pull request by imankit**

### CODE REVIEW

File: app/components/app.js [hunk 1]
```diff
@@ -24,18 +24,20 @@ class App extends React.Component {
...
-                            left={70}
-                            top={0}
-                            status="loading"
-                            className="loadermain"
-                        />
-                        :
-                        this.props.children
-                }
+                <div>
+                    {this.props.children}
```
Comment (shubhamqweasd) [inline app/components/app.js]: this is wrong , change this back.

Comment (imankit) [inline app/components/app.js]: Previous code refreshes the page when uploading an image

Comment (shubhamqweasd) [inline app/components/app.js]: yes , there are other ways to stop that , this is not the correct way

Comment (imankit) [inline app/components/app.js]: Ok I'll do it

Comment (shubhamqweasd) [inline app/components/app.js]: we have three level of loader , you can use them

**Pull request merged**